src/python/lib/ensembl/compara/citest/testfiles.py

```diff
@@ -0,0 +1,144 @@
+"""
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import filecmp
+import os
+from typing import Dict, List, Union
+
+import pytest
+from _pytest._code.code import ExceptionChainRepr, ExceptionInfo, ReprExceptionInfo
+from _pytest.fixtures import FixtureLookupErrorRepr
+
+from compara.utils import DirCmp, PathLike, to_list
```

Comment:
Can that be changed to a relative import ? `compara.` is not supposed to exist

Reply:
Based on that comment and what we discussed yesterday, I think it may be safer to just do relative imports for everything (although I'm not a fan, tbh).